packages/aila/src/protocol/schema.ts

```diff
@@ -646,3 +649,13 @@ export const chatSchemaWithMissingMessageIds = z
 export type AilaPersistedChatWithMissingMessageIds = z.infer<
   typeof chatSchemaWithMissingMessageIds
 >;
+
+export type LessonPlanSectionWhileStreaming =
```

Comment:
Add a type to represent all the different sections that could be part of a lesson plan